--- gcc-4.8.5/configure	2015-05-03 10:26:29.000000000 -0700
+++ gcc-4.8.5/configure	2016-07-30 09:38:49.719444355 -0700
@@ -3794,6 +3794,10 @@
     ;;
 esac
 
+# If we are building against NuttX, then don't attempt to build libstdc++
+# (should be conditioned on --with-nuttx)
+noconfigdirs="$noconfigdirs target-libiberty target-libstdc++-v3"
+
 # If we aren't building newlib, then don't build libgloss, since libgloss
 # depends upon some newlib header files.
 case "${noconfigdirs}" in
